thirdparty/jquery-entwine/jquery.entwine.js

```diff
@@ -1157,148 +1117,117 @@ catch (e) {
 	});
 	
 })(jQuery);
-;
 
 
 /* src/domevents/jquery.entwine.domevents.addrem.js */
 
 (function($){
 
-	// Gets all the child elements of a particular elements, stores it in an array
-	function getElements(store, original) {
-		var node, i = store.length, next = original.firstChild;
+	/** Recursively gets all the descendent elements of a particular element and stores it in an array */
+	function addNodeChildrenToList(ancestor, list) {
+		var node, i = list.length, next = ancestor.firstChild;
 
 		while ((node = next)) {
-			if (node.nodeType === 1) store[i++] = node;
+			if (node.nodeType === 1) list[i++] = node;
 			next = node.firstChild || node.nextSibling;
-			while (!next && (node = node.parentNode) && node !== original) next = node.nextSibling;
+			while (!next && (node = node.parentNode) && node !== ancestor) next = node.nextSibling;
 		}
 	}
 
-	// This might be faster? Or slower? @todo: benchmark.
-	function getElementsAlt(store, node) {
-		if (node.getElementsByTagName) {
-			var els = node.getElementsByTagName('*'), len = els.length, i = 0, j = store.length;
-			for(; i < len; i++, j++) {
-				store[j] = els[i];
-			}
+	/** Handles a list of added or removed nodes from a mutation observer */
+	function handleMutation(list, eventName) {
+		if (list.length === 0) {
+			return;
 		}
-		else if (node.childNodes) {
-			var els = node.childNodes, len = els.length, i = 0;
-			for(; i < len; i++) {
-				getElements(store, els[i]);
+		// Get all mutated elements.
+		const mutated = [];
+		for (let node of list) {
+			if (node.nodeName === '#text') {
+				continue;
 			}
+			mutated.push(node);
+			addNodeChildrenToList(node, mutated);
 		}
-	}
-
-	var dontTrigger = false;
-
-	var patchDomManipCallback = function(original) {
-		var patched = function(elem){
-			var added = [];
-
-			if (!dontTrigger) {
-				if (elem.nodeType == 1) added[added.length] = elem;
-				getElements(added, elem);
-			}
-
-			var rv = original.apply(this, arguments);
 
-			if (!dontTrigger && added.length) {
-				var event = $.Event('EntwineElementsAdded');
-				event.targets = added;
-				$(document).triggerHandler(event);
-			}
-
-			return rv;
+		if (mutated.length === 0) {
+			return;
 		}
-		patched.patched = true;
-
-		return patched;
-	}
 
-	var version = $.prototype.jquery.split('.');
-	var callbackIdx = (version[0] > 1 || version[1] >= 10 ? 1 : 2);
-
-	// Monkey patch $.fn.domManip to catch all regular jQuery add element calls
-	var _domManip = $.prototype.domManip;
-	$.prototype.domManip = function() {
-		if (!arguments[callbackIdx].patched) arguments[callbackIdx] = patchDomManipCallback(arguments[callbackIdx]);
-		return _domManip.apply(this, arguments);
-	}
-
-	// Monkey patch $.fn.html to catch when jQuery sets innerHTML directly
```

Comment:
This was always a dangerous way to detect this - but thankfully it isn't necessary because setting `innerHTML` will result in new nodes in the mutation observer.